pkg/scheduler/ugm/manager.go

```diff
@@ -607,6 +607,35 @@ func (m *Manager) Headroom(queuePath, applicationID string, user security.UserGr
 	return resources.ComponentWiseMinPermissive(userHeadroom, groupHeadroom)
 }
 
+// CanRunApp checks the maxApplications for this specific application that runs as the user and group.
+func (m *Manager) CanRunApp(queuePath, applicationID string, user security.UserGroup) bool {
+	userTracker := m.getUserTracker(user.User)
```

Comment:
Does userTracker always exists? I think this is going to be the first entry point into ugm module as this is being called from the very early steps.

Reply:
yes, correct